Modelica/Blocks/Interfaces.mo

```diff
@@ -1516,7 +1507,6 @@ of noise blocks.
               extent={{-18,-10},{18,-30}},
               textString="%Name_fder2",
               visible=(use_fder and use_fder2))}),
-              Diagram(coordinateSystem(preserveAspectRatio=false)),
```

Comment:
I don't see why this should be removed. To me it makes sense to have icon and diagram coordinate systems "in sync" and thus they should have the same preserveAspectRatio.

Reply:
Can you explain what practical function `preserveAspectRatio=false` serves in this case where there is no diagram defined?

Reply:
Just tested it and it has no effect since there is no diagram defined. So it can be seen as superfluous noise.